backend/strapi-sync/sync.js

```diff
@@ -258,33 +259,29 @@ export async function syncFeatureTypes(strapiData, featureTypeData) {
 
 /**
  * These are not associated with any specific model in Strapi
- * We will create these manually once
- * @returns {Promise[Object]} resolves when all dateTypes have been created
+ * We will create these manually
+ * @param {Object} item dateType data from db
+ * @returns {DateType} dateType model
```

Comment:
Since it's an async function, it technically returns a Promise that resolves to a DateType

I think there's a mix of angle brackets and square brackets in this project, but it's mostly angle brackets so let's go with that here:

```suggestion
 * @returns {Promise<DateType>} dateType model
```